client/galaxy/scripts/entry/panels/admin-panel.js

```diff
@@ -41,6 +41,12 @@ const AdminPanel = Backbone.View.extend({
                         target: "__use_router__",
                         id: "admin-link-jobs"
                     },
+                    {
+                        title: _l("Manage workflow invocations"),
```

Comment:
```suggestion
                        title: _l("Workflow invocations"),
```

Reply:
Crap - sorry I missed this - I'll fix this in a follow up.